scripts/installscripts/buster-install-default.sh

```diff
@@ -327,7 +334,7 @@ check_existing() {
 
     # The install will be in the home dir of user pi
     # Move to home directory now to check
-    cd ~ || exit
+    cd "${home_dir}"
```

Comment:
I think this is not your fault, but using now the same name for a local variable is confusing.

I would rename it within the function to `local_home_dir`or something similar

Reply:
i just used the already existing variable `home_dir` to get rid of `~`. 

But i get your point. I renamed it to `local_home_dir`